src/Dapr.Client/DaprClient.cs

```diff
@@ -229,6 +229,35 @@ public async ValueTask<StateEntry<TValue>> GetStateEntryAsync<TValue>(string sto
             Dictionary<string, string> metadata = default,
             CancellationToken cancellationToken = default);
 
+        /// <summary>
+        /// Saves the provided <paramref name="stateTransactionRequests" /> to the Dapr state
+        /// store.
+        /// </summary>
+        /// <param name="storeName">The name of the state store.</param>
+        /// <param name="stateTransactionRequests">A list of StateTransactionRequests.</param>
+        /// <param name="metadata">An key/value pair that may be consumed by the state store.  This depends on the state store used.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken" /> that can be used to cancel the operation.</param>
+        /// <returns>A <see cref="ValueTask" /> that will complete when the operation has completed.</returns>
+        public abstract Task ExecuteStateTransactionAsync(
+            string storeName,
+            IReadOnlyList<StateTransactionRequest> stateTransactionRequests,
```

Comment:
I think this name is fine, but throwing an idea out there - maybe it should be called operations like in the api spec?